front-end/package.json

```diff
@@ -17,10 +17,10 @@
     "styled-components": "^5.1.0"
   },
   "scripts": {
-    "start": "PORT=3001 react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "cross-env NODE_ENV=development PORT=3001 react-scripts start",
```

Comment:
Similiar to old PR, remove any cross-env and NODE_ENV where not really needed

Reply:
Updated